index.html

```diff
@@ -26,9 +26,9 @@ <h1 class="logo"><a href="#">My Logo</a></h1>
                     fill="#6070FF" />
             </svg>
             <ul class="nav-links">
-                <li class="nav-link"><a href="#">Portfolio</a></li>
-                <li class="nav-link"><a href="#">About</a></li>
-                <li class="nav-link"><a href="#">Contact</a></li>
+                <li class="nav-link">Portfolio</a></li>
+                <li class="nav-link">About</a></li>
+                <li class="nav-link">Contact</a></li>
```

Comment:
- [ ] You have done well to apply the required changes by the previous reviewer on the nav links 👏 👏 . However, you should note that the nav links should be inside `a` tag such that when clicked you are taken to the respective section. Kindly ensure that you apply this to have a professional portfolio ☺️ . Read more here on [geegsforgeegs](https://www.geeksforgeeks.org/how-to-create-links-to-sections-within-the-same-page-in-html/)